main.go

```diff
@@ -413,11 +412,11 @@ func outputToJSON(issues chan *Issue) int {
 	return status
 }
 
-func runLinters(linters map[string]*Linter, paths, ellipsisPaths []string, concurrency int, exclude *regexp.Regexp, include *regexp.Regexp) (chan *Issue, chan error) {
```

Comment:
sounds good. I've made that change.

I think it would be possible to remove the global `config` entirely and pass around a reference, but I'm not sure if you would be interested in such a change.